### PR TITLE
Nightwatch Test Uplift

### PR DESCRIPTION
**Pull request by emilyrohrbough**
### Summary
This PR is purely to test the recently proposed toolkit changes.

@cerner/terra

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,14 +52,14 @@
     "lint": "npm run lint:js && npm run lint:scss",
     "lint:js": "eslint --ext .js,.jsx .",
     "lint:scss": "lerna run lint:scss",
-    "pretest": "npm run lint",
+    "pretest-disabe": "npm run lint",
```
Comment (mjhenkes) [inline package.json]: I assume you meant 'pretest-disable'

File: clinical-nightwatch-changes.txt [hunk 2]
```diff
@@ -0,0 +1,2345 @@
+
```
Comment (bjankord) [inline clinical-nightwatch-changes.txt]: what is this file?

File: clinical-nightwatch-original.txt [hunk 3]
```diff
@@ -0,0 +1,1916 @@
+
```
Comment (bjankord) [inline clinical-nightwatch-original.txt]: what is this file?

File: clinical-nightwatch-changes.txt (hunk 2 above)
Comment (emilyrohrbough) [reply clinical-nightwatch-changes.txt]: Oh. These should not have been included. It was from when I was doing timing comparisons

Comment (bjankord) [reply clinical-nightwatch-changes.txt]: ah ok, makes sense.

**Comment by emilyrohrbough** [comment]
Resolves https://github.com/cerner/terra-clinical/issues/181

**Pull request merged**